src/SingleBookingPage/bookingItem/AccordionLegCitiesInfo.js

```diff
@@ -139,19 +137,12 @@ const LegCitiesInfo = (props: Props) => {
           </div>
           <div className="infoRow">
             <InformationCircle size="small" customColor="#00a991" />
-            <span className="moreInfoLink">
-              <TextLink
-                url=""
-                onClick={e => {
-                  e.preventDefault();
-                  props.history.push(
-                    '/faq/search/article/RkFRQXJ0aWNsZToxMjc=',
-                  );
-                }}
-                size="small"
-                title="Trains and Buses Info"
-              />
-            </span>
+            <Link
```

Comment:
I have changed to Link here, as Orbit TextLink is supposed to be used in paragraphs for a part of the text, which is not the case here.